devnet/eth-devnet.yaml

```diff
@@ -57,18 +54,12 @@ spec:
           command:
             - /bin/sh
             - -c
-            - "npm run migrate && npx truffle exec scripts/deploy_test_token.js && npx truffle exec scripts/register_all_chains.js && npm run deploy-relayers-evm1 && nc -lkn 2000"
+            - "CHAIN_ID=2 NETWORK=devnet ./forge-scripts/devnetInitialization.sh"
```

Comment:
This `CHAIN_ID` would be the wormhole chain id, right? I got confused why it differs from the one above but that would make sense.

Reply:
yup, wormhole chain id